### PR TITLE
Raised PR for aws-template

### PR DESCRIPTION
**Pull request by priyadishah**
Hi @robe070 

Raising this PR as this branch have multiple modifications including value of 2022 in AMI Mapping, Tagging values, path values etc.

please review and let me know if I need to make any changes.

### CODE REVIEW

File: scripts/GitRepoWriteAccess.ps1 [hunk 1]
```diff
@@ -25,10 +25,14 @@ param (
   )
 
 # goto git repo
+<<<<<<< HEAD
+cd "$($env:Pipeline_Workspace)/$($GitRepoName)"
+=======
```
Comment (robe070) [inline scripts/GitRepoWriteAccess.ps1]: fix this

File: scripts/GitRepoWriteAccess.ps1 [hunk 2]
```diff
@@ -25,10 +25,14 @@ param (
...
 # cd "$($env:System_DefaultWorkingDirectory)/$($GitRepoName)"
 
 # Changes path from env:System_DefaultWorkingDirectory to $env:Pipeline_Workspace
 cd "$($env:Pipeline_Workspace)/$($GitRepoName)" 
+>>>>>>> 3a39f87b56f6073f63a03f9b10378485d6ff4137
```
Comment (robe070) [inline scripts/GitRepoWriteAccess.ps1]: fix this

**Review by robe070 — CHANGES_REQUESTED** [review]
Hasn't somebody else made almost all these changes and raised another PR for it?

File: scripts/GitRepoWriteAccess.ps1 (hunk 1 above)
Comment (priyadishah) [reply scripts/GitRepoWriteAccess.ps1]: done, forgot to commit.

File: scripts/GitRepoWriteAccess.ps1 (hunk 2 above)
Comment (priyadishah) [reply scripts/GitRepoWriteAccess.ps1]: done, forgot to commit.

**Pull request merged**